rules/react/walmart.js

```diff
@@ -35,8 +45,12 @@ module.exports = {
     "react/no-did-mount-set-state": 2,
     // Prevent usage of setState in componentDidUpdate
     "react/no-did-update-set-state": 2,
+    // Prevent direct mutation of this.state
+    "react/no-direct-mutation-state": 0,
     // Prevent multiple component definition per file
     "react/no-multi-comp": 0,
```

Comment:
This is already in most of the apps via bolt so looks like not. I think it probably should be though. Might be a lot of work to flip this switch. Should we track this in an issue?